_pkgdown.yml

```diff
@@ -114,6 +114,7 @@ articles:
     navbar: ~
     contents:
       - tidyverse_translation
+      - summary_selecting
```

Comment:
Not sure I find this filename to be that descriptive. How about "selection_syntax.Rmd"?